### ISSUE TITLE
reproduce colab finetuning

### ISSUE THREAD

**Issue by YHYeooooong** [google-research/simclr#138]
Hi! thanks to share your good work!
I have a short question about fine-tuning code

I have been trying to reproduce the result of colab(tf2) fine-tuning code by using the original code in Github(tf1). 
But I got the different result with the following commands line : 

```
python run.py --mode=train_then_eval --train_mode=finetune \
  --fine_tune_after_block=4 --zero_init_logits_layer=True \
  --variable_schema='(?!global_step|(?:.*/|^)Momentum|head)' --train_summary_steps=1 \
  --global_bn=False --optimizer=lars --learning_rate=0.0001 --weight_decay=0.0 \
  --train_epochs=300 --train_batch_size=10 --warmup_epochs=0 \
  --dataset=cifar10 --image_size=224 --eval_split=test --resnet_depth=152 \
  --checkpoint=../model/r152_1x_sk0_tf1 --model_dir=./trained_model/simclr152_lineareval_lars_bat10_sumstep1 --use_tpu=False
```

On colab, I used 300 epochs with 10 batch sizes, with my own dataset(224*224), and the resnet152 pre-trained model.
Also, I use a lars optimizer with a learning rate of 0.0001, weight decay of 0.
I think there is something to handled like the learning rate scale, But I don't know which value to modify

Can you give me a default setting of colab finetuning code, or some guidelines?

**Comment by chentingpc** [comment]
You may want to increase batch size and tune the learning rate a bit. Consider lowering training epoch and/or using a smaller resnet for testing.

**Comment by YHYeooooong** [comment]
Thanks for your reply!
I'll do as you advise. I have another question.
In fact, while I was struggling, I found a code for the learning rate scheduler here in Github.

https://github.com/google-research/simclr/blob/3ad6700c1b139ee18e43f73546b7263a710de699/model_util.py#L59
 
But you can't find it in the collab fine-tuning code.
Is there a difference in settings such as the learning scheduler between the Github fine-tuning code and the Colab fine-tuning code similar to the previous one?

**Comment by chentingpc** [comment]
learning rate scaling only matters when you wanted to scale between different batch size. Set it to linear if you use sgd/momentum, sqrt if you use lars optimizer.

**Comment by YHYeooooong** [comment]
Thanks! I'll try!

**Comment by deepankarvarma** [comment]
@YHYeooooong can you share the contents of --checkpoint and the --model_dir
